### PR TITLE
CMS integration parameter for Jenkins

### PR DESCRIPTION
**Pull request by wyattwalter**
## Acceptance criteria
- [ ]

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

ref: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16422

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -14,6 +14,12 @@ VAGOV_BUILDTYPES = [
   'vagovprod'
 ]
 
+DRUPAL_MAPPING = [
```
Comment (wyattwalter) [inline Jenkinsfile]: I'm not sure if this is quite settled yet, but easy enough to update.. I would need to test, but if staging drupal ended up mapped to our dev and staging, we could just pass in a list in this dictionary.

**Comment by wyattwalter** [comment]
@jbalboni This adds a parameter, but does _not_ do anything to optimize build times based upon previous bundles from webpack yet.

I'm pretty confident there's a race condition here where if a commit hits master and then this job gets triggered with the parameter for a specific environment, the job from the commit would take a bit longer and would upload last -- meaning that the change might not show up initially. Not sure if that's super critical, but I think we could deal with it by limiting concurrency if needed.

**Comment by jbalboni** [comment]
@wyattwalter I think you're right, we'll probably want to limit the number of these jobs running at once to get around that.

File: Jenkinsfile (hunk 1 above)
Comment (jbalboni) [reply Jenkinsfile]: I think this is what we want.

**Review by jbalboni — APPROVED** [review]
Looks good to me, had one minor question.

**Pull request merged**